library/ZendService/Apple/Apns/Message.php

```diff
@@ -129,7 +129,7 @@ public function setToken($token)
             ));
         }
 
-        if (preg_match('/[^0-9a-f]/', $token)) {
+        if (preg_match('/[^0-9a-fA-F]/', $token)) {
```

Comment:
Easier fix is to use the `/i` flag on the regex; I'll make that change on merge.